src/zigwin32.build.zig

```diff
@@ -1,16 +1,27 @@
 const builtin = @import("builtin");
 const std = @import("std");
 
-const is_zig_0_11 = std.mem.eql(u8, builtin.zig_version_string, "0.11.0");
+const zig_version = builtin.zig_version;
 
 pub fn build(b: *std.Build) void {
-    if (is_zig_0_11) {
-        _ = b.addModule("zigwin32", .{
-            .source_file = .{ .path = "win32.zig" },
-        });
-    } else {
-        _ = b.addModule("zigwin32", .{
-            .root_source_file = .{ .path = "win32.zig" },
-        });
+    switch (zig_version.minor) {
+        11 => {
+            _ = b.addModule("zigwin32", .{
+                .source_file = .{ .path = "win32.zig" },
+            });
+        },
+        12 => {
+            _ = b.addModule("zigwin32", .{
+                .root_source_file = .{ .path = "win32.zig" },
+            });
+        },
+        13 => {
```

Comment:
Let's make this the else, there's a non zero chance the zigwin32 code today will work with future zig versions, it's just bindings and a few lines of build.zig.

Reply:
Gotcha, sounds good!